app/modules/users/resources.py

```diff
@@ -136,15 +136,11 @@ def _process_patch_operation(self, operation, user, state):
         Returns:
             processing_status (bool) - True if operation was handled, otherwise False.
         """
-        if 'value' not in operation:
-            # TODO: handle errors better
-            abort(code=http_exceptions.UnprocessableEntity.code, message="value is required")
-
         if operation['op'] == parameters.PatchUserDetailsParameters.OP_TEST:
             # User has to provide the admin/supervisor password (if the current
             # user has an admin or a supervisor role) or the current password
             # of the user that is edited.
-            if operation['path'] == '/current_password':
+            if operation['path'] == 'current_password':
```

Comment:
This is why I think we need another name instead of `path`. `path` is something with slash and here we don't have one.